src/edu/buffalo/cse/jive/finiteStateMachine/util/ShortestPathHolderForEExpression.java

```diff
@@ -0,0 +1,23 @@
+package edu.buffalo.cse.jive.finiteStateMachine.util;
+
+import java.util.List;
+
+import edu.buffalo.cse.jive.finiteStateMachine.models.State;
+
+public class ShortestPathHolderForEExpression {
+	
+	public static List<Pair<State, State>> path;
+
+	public static List<Pair<State, State>> getPath() {
+		return path;
+	}
+
+	public static void setPath(List<Pair<State, State>> path) {
+		ShortestPathHolderForEExpression.path = path;
+	}
+	
```

Comment:
This static class holds the shortest path evaluated from EExpression class. It is getting used when we generate the visualization and marks the green arrow between states.